docs/data/stashcache/install-origin.md

```diff
@@ -100,11 +102,11 @@ The origin service consists of the following systemd units:
 
 | **Software** | **Service name** | **Notes** |
 |--------------|------------------|-----------|
-| XRootD | `xrootd@stashcache-origin-server.service` | The xrootd daemon, which performs the data transfers |
-| XRootD | `cmsd@stashcache-origin-server.service` | The "cluster management service" daemon, which integrates the origin into the data federation.  |
-| Fetch CRL         | `fetch-crl-boot` and `fetch-crl-cron` | See [CA documentation](/common/ca#managing-fetch-crl-services) for more info |
+| XRootD | `xrootd@stash-origin.service` | The xrootd daemon, which performs the data transfers |
+| XRootD | `xrootd@stash-origin-auth.service` | The xrootd daemon, which performs authenticated data transfers |
+| XRootD | `cmsd@stash-origin.service` | The "cluster management service" daemon, which integrates the origin into the data federation.  |
```

Comment:
Do we need a `cmsd@stash-origin-auth.service` as well?

Reply:
Nope - the `cmsd` should export both.

(We might want to revisit this, however... this implies that the `xrootd@stash-origin` is running)

Reply:
So something like
```
!!! note
    Make sure `xrootd@stash-origin.service` is running before starting `cmsd@stash-origin.service`
```

Reply:
And tweaking the description for cmsd:  The "cluster management service" daemon, which integrates the xrootd daemon(s) into the data federation